docs/source/examples.md

```diff
@@ -274,3 +274,87 @@ By default, the `hi-ml` package assumes that you supply a commandline argument `
 the commandline) to trigger a submission of the present script to AzureML. If you wish to control it via a different
 flag, coming out of your own argument parser, use the `submit_to_azureml` argument of the function
 `health.azure.himl.submit_to_azure_if_needed`. 
+
+## Training with k-fold cross validation in Azure ML
+To train with k-fold cross validation, you must do two things.
```

Comment:
Maybe introduce the concept of a parent run and child runs here first, pointing to AML docu?